packages/ui/src/pages/About/AboutPage.tsx

```diff
@@ -0,0 +1,62 @@
+/*
+    Copyright (C) 2024 Red Hat, Inc.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+            http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+*/
+import { Bullseye, Content, Timestamp, TimestampFormat, TimestampTooltipVariant } from '@patternfly/react-core';
+import { FunctionComponent, useMemo } from 'react';
+import logo from '../../assets/logo-kaoto-dark.png';
+import { GIT_DATE, GIT_HASH, KAOTO_VERSION } from '../../version';
+import './AboutPage.scss';
+
+export const AboutPage: FunctionComponent = () => {
+  const buildDate = useMemo(() => new Date(GIT_DATE), []);
+  const TOOLTIP_PROPS = { variant: TimestampTooltipVariant.default } as const;
+
+  return (
+    <Bullseye className="about-page">
+      <img className="about-page-icon" src={logo} alt="Kaoto icon" />
```

Comment:
shouldn't the alt more precise and mention that it is the about page icon

Reply:
I'll take care in my PR https://github.com/tplevko/kaoto-next/pull/27